src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -53,6 +53,10 @@
   visibility: hidden;
 }
 
+.player .on-top-ad .bottom-bar{
+  width: auto;
```

Comment:
check if can be added to `adGuiWrapper` class

Reply:
cannot